src/ngraph/runtime/gpu/unit_test.manifest

```diff
@@ -34,6 +34,9 @@ max_pool_3d
 avg_pool_3d
 argmin_trivial
 argmax_trivial
+argmax_3D_axis_0
```

Comment:
You are disabling these new tests. Do they not pass? Do they pass for all other backends?